README.md

```diff
@@ -99,10 +91,33 @@ type StoryPaths = {|
 |}
 
 // Retrieve and group the different stories based on their filenames
-function getFiles(options: Options): Array<StoryPaths>
+function getFiles({|
+  baseUrl?: string,              // The url for the story, defaults to:
+                                 //   'http://localhost:6006'
+  flattenFolders?: Array<string> // Folders to flatten
+                                 //   defaults to ['__snapshots__', '__screenshots__']
+  stories: any,                  // Output of require context with the stories
```

Comment:
do we have no insight as to the shape of this?

Reply:
we do, I just don't know how to type it. It's a function that also has additional methods (like `keys()`). Since it's not something anyone writes (it's the output of `require.context`) it didn't seem a big deal have it untyped